.github/workflows/release-chart.yaml

```diff
@@ -22,39 +22,58 @@ jobs:
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      - name: Get chart version
+        id: chart_version
+        uses: mikefarah/yq@v4.19.1
+        with:
+          cmd: yq eval '.version' './charts/metrics-server/Chart.yaml'
+
+      - name: Get chart app version
+        id: chart_app_version
+        uses: mikefarah/yq@v4.19.1
+        with:
+          cmd: yq eval '.appVersion' './charts/metrics-server/Chart.yaml'
+
+      - name: Check release
```

Comment:
What does check release really checks? If the release_version is valid, non empty?

Reply:
It currently checks that the chart doesn't have a pre-release version suffix, if it does it means that it shouldn't be released. This is how we can update _Chart.yaml_ on `master` to be used by e2e tests but not have the chart released when the release branch is cut.

I'm also going to add a check that the app version is for the current release branch for full safety.

I'll change some of the naming to make this clearer.

Reply:
This has been done and pushed.